### PR TITLE
Don't bundle in tests folder in lib

### PR DESCRIPTION
**Pull request by sunwrobert**
Decreases the unpacked size from 1.2 MB -> 850KB

### CODE REVIEW

**Review by Meemaw — COMMENTED** [review]
My concern is that since we publish locally, one could still do a mistake and include it. Wdyt about adding it to npm ignore?

File: ts-config-build.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (Meemaw) [inline ts-config-build.json:1]: **nit:** lets name `tsconfig.build.json`.

**Review by Meemaw — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
   ],
   "scripts": {
     "abi-type-gen": "yarn run typechain --target=web3-v1 src/abi/*.json --out-dir=src/typechain/contracts",
-    "build": "yarn abi-type-gen && tsc",
+    "build": "yarn abi-type-gen && tsc -p ts-config.build.json",
```
Comment (Meemaw) [inline package.json:21]:
```suggestion
    "build": "yarn abi-type-gen && tsc -p tsconfig.build.json",
```

Comment (sunwrobert) [reply package.json:21]: oh woops

**Review by Meemaw — APPROVED** [review]

**Pull request merged**